metadata/metadata.ts

```diff
@@ -3183,6 +3183,14 @@ export const OPEN_SEA_ITEMS: Record<InventoryItemName, Metadata> = {
     image_url: "../public/erc1155/images/453.png",
     attributes: [],
   },
+  "Genie Lamp": {
+    description:
+      "A magical lamp that contains a genie who will grant you three wishes.",
+    decimals: 0,
+    external_url: "https://docs.sunflower-land.com/getting-started/about",
+    image_url: "../public/erc1155/images/460.png",
```

Comment:
For the `460.png`, we will want to increase the size to 1920x1920px.

I've found that is the perfect size for Open Sea that it loads quick + isn't blurry.

Also consider layering on top the `boost.png` since it has utility

Reply:
Will do @adamhannigan !